tomography_preprocessing/denoising/care4relion/care4relion.py

```diff
@@ -0,0 +1,149 @@
+from pathlib import Path
+from typing import Optional
+
+import pandas as pd
+import starfile
+import rich
+import typer
+import subprocess
+from rich.progress import track
+
+from .train_neural_network import train_neural_network
+from ._utils import *
+from .constants import predict_config_prefix, even_suffix 
+from .._cli import cli
+from ...utils.relion import relion_pipeline_job
+
+console = rich.console.Console(record=True)
+
+@cli.command(name='Care4Relion')
```

Comment:
```suggestion
@cli.command(name='cryoCARE')
```

general rename suggestion

Reply:
Thought Care4Relion was funny ;) will change tho!